### DISCUSSION TITLE
flask-sock ssl

### DISCUSSION THREAD

**Discussion by zimsekdanilo** [miguelgrinberg/flask-sock#90] [Q&A]
How do I deploy flask-scok application with ssl? What changes should be made?

**Comment by miguelgrinberg** [comment]
This package is not affected. You just need to configure your web server to use SSL.

**Comment by ilirosmanaj** [comment]
@miguelgrinberg, does this mean that the websocket will automatically use the web server configuration of using SSL?

**Comment by miguelgrinberg** [comment]
SSL decryption and encryption is done by the web sierver to the incoming and outgoing traffic. This package does not need to know what happens to the traffic, it always works with the raw data.
